error_test.go

```diff
@@ -38,3 +38,55 @@ func TestErrorResponse(t *testing.T) {
 	assert.Equal(t, "req_123", stripeErr.RequestID)
 	assert.Equal(t, 401, stripeErr.HTTPStatusCode)
 }
+
+func TestErrorRedact(t *testing.T) {
+	pi := &PaymentIntent{Amount: int64(400), ClientSecret: "foo"}
+	si := &SetupIntent{Description: "keepme", ClientSecret: "foo"}
+
+	// Both
+	{
```

Comment:
This isn't quite a standard yet because I thought about it too late, but consider changing these blocks to subtests like:

``` go
t.Run("BothIntentObjects", func(t *testing.T) {
...
```

This is a little nicer because it means that you can now target one specific subtest to run with Go test:

``` sh
go test . -run BothIntentObjects
```